life.bolls.bolls.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: life.bolls.bolls
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: bolls
+separate-locales: false
+tags: [proprietary]
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
```

Comment:
given the scope of the app - what do you need full home write access for?

Reply:
To save configs and download Bibles for offline. This app is an electron application.

Reply:
I don't think you need home access for that. Why not XDG_CONFIG_HOME? I'm pretty sure electron has an api to get that.

Reply:
ok, how to change that to XDG_CONFIG_HOME?

Reply:
You just need to tell your electron app to store these files in 
```
const { app } = require ('electron');
const userDataPath = app.getPath ('userData');
```
you can remove the home permission later.

Reply:
I have changed the `--filesystem=~/.config`. Is it enough?